install-1x.html

```diff
@@ -322,17 +322,17 @@ <h4>8. Other INI files</h4>
         <h2>2. Install cypht using Docker</h2>
         <p>
             Using Docker is one of the easiest way to install cypht as the cypht docker image comes with all the steps
-            required in the manual instalation done for you. But the bad news is that this instllation way requires
+            required in the manual instalation done for you. But the bad news is that this installation way requires
             docker knowledge.<br />
             Here is the cypht docker repository: <a
                 href="https://hub.docker.com/r/sailfrog/cypht-docker">https://hub.docker.com/r/sailfrog/cypht-docker</a><br />
```

Comment:
As in _install.html_ put:
`https://hub.docker.com/r/cypht/cypht` instead of https://hub.docker.com/r/sailfrog/cypht-docker.

Reply:
r/cypht/cypht only has [Cypht 2](https://hub.docker.com/r/cypht/cypht/tags)! 

While Sailfrog has no version info, the latest release of 10 months ago tracks with v2 only releasing in April (6 months ago)

Reply:
Well... Thanks. You are right.